tests/Fixtures/ConvertConfig/extensions.yaml

```diff
@@ -0,0 +1,8 @@
+default:
```

Comment:
We have a number of yaml files that we use to test all the different cases that we want to test